backend/services/user.go

```diff
@@ -480,26 +481,11 @@ func UpdateUserProfile(ctx context.Context, db *database.Connection, i UpdateUse
 	return nil
 }
 
-// DeleteSessionsForUserSlug finds all existing sessions for a given user, then removes them, effectively
-// logging the user out of the service.
-func DeleteSessionsForUserSlug(ctx context.Context, db *database.Connection, userSlug string) error {
```

Comment:
This function was only getting called in a test, and isn't really usable with our new setup, so I deleted it